server/gameconnection.py

```diff
@@ -210,41 +219,32 @@ async def handle_desync(self, *_args):  # pragma: no cover
         self.game.desyncs += 1
 
     async def handle_game_option(self, key, value):
-        if key == 'Victory':
-            self.game.gameOptions['Victory'] = Victory.from_gpgnet_string(value)
-        elif key in self.game.gameOptions:
-
-            """
-            This block about AIReplacement is added because of a mistake in the FAF game patch code
-            that makes "On" and "Off" be "AIReplacementOn" and "AIReplacementOff". The code
-            below removes that extra statement to make it a simple "On" "Off".
-            This block can be removed as soon as the game sends "On" and "Off" instead of
-            "AIReplacementOn" and "AIReplacementOff" to the server as game options.
-            https://github.com/FAForever/fa/issues/2610
-            """
-            if key == "AIReplacement":
-                value = value.replace("AIReplacement", "")
+        if not self.is_host():
+            return
 
+        if key == "Victory":
+            self.game.gameOptions["Victory"] = Victory.from_gpgnet_string(value)
+        else:
             self.game.gameOptions[key] = value
 
         if key == "Slots":
             self.game.max_players = int(value)
-        elif key == 'ScenarioFile':
+        elif key == "ScenarioFile":
             raw = "%r" % value
```

Comment:
I find the following easier to read:
```suggestion
            raw = repr(value)
```
I am also not sure why `repr` instead of `str` - is `value` ever anything else but a string or a number?

Reply:
I have no idea. Possibly a dict?